Tests/AppTests/Mocks/Search.Result+mock.swift

```diff
@@ -12,12 +12,51 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+import Foundation
```

Comment:
Nit: Doesn't look like this import is needed, I think Xcode pulled that in when it was `Date(...)` instead of `.t0` below.